docs/antora/modules/ROOT/pages/management/docker.adoc

```diff
@@ -204,10 +217,126 @@ If you wish to take a snapshot of your Docker installation, the details can be f
+
+For `docker`, to check the logs for the different microservices of Lisk Service, use the command `docker container logs CONTAINER`, where `CONTAINER` is the respective Docker container that holds the logs you wish to view.
 
 For example, to see the logs for the Gateway microservice, execute the following command:
 
 ----
 docker container logs lisk-service_gateway_1
 ----
+
+To check the logs for different microservices using the `make` commands, the following commands listed below can be used:
+
+.Displays the logs for all the microservices that have been logged so far, and that are currently existing
+----
+make logs
+----
+
+.Displays the latest logs for all the microservices
+----
+make logs-live
+----
+
+.Displays the logs for the specified microservice.
+----
+make logs-blockchain-connector
+----
+In the above example, it will display logs only for the `blockchain-connector` microservice.
+
+.Displays the latest logs for the specified microservice.
+----
+make logs-live-blockchain-connector
+----
+In the above example, it will display latest logs only for the `blockchain-connector` microservice.
+
+
+Replace the `blockchain-connector` with the specific service name required.
+
+
+== Upgrade Lisk Service
+
+To upgrade the Docker container to a desired version, please follow one of the two options below. To find all the tagged versions for Lisk Service, please check the available tags (https://github.com/LiskHQ/lisk-service/tags) on GitHub.
+
+==== Option A - Download pre-built images from DockerHub
+
+. Update `docker-compose.yml` to download the specific image versions from the DockerHub as shown below. Update the `0.7.0` tag against the specified images with the desired version.
++
+----
+blockchain-app-registry:
+    image: lisk/service_blockchain_app_registry:0.7.0
+    ...
+
+  blockchain-connector:
+    image: lisk/service_blockchain_connector:0.7.0
+    ...
+
+  blockchain-indexer:
+    image: lisk/service_blockchain_indexer:0.7.0
+    ...
+
+  blockchain-coordinator:
+    image: lisk/service_blockchain_coordinator:0.7.0
+    ...
+
+  transaction-statistics:
+    image: lisk/service_transaction_statistics:0.7.0
+    ...
+
+  fee-estimator:
+    image: lisk/service_fee_estimator:0.7.0
+    ...
+
+  gateway:
+    image: lisk/service_gateway:0.7.0
+    ...
+
+  market:
+    image: lisk/service_market:0.7.0
+    ...
+
+  export:
+    image: lisk/service_export:0.7.0
+
+----
+
+. Stop Lisk Service
++
+----
+make stop
+----
+NOTE: In case the database needs to be flushed please execute the `make down` command instead. Please check the {url_rel_notes}[GitHub release notes] for the relevant version to verify if the existing databases need to be flushed.
+
+. Start Lisk Service in the containers.
++
+----
+make up
+----
+
+
+==== Option B - Build images locally
+
+Build the images locally using the following steps.
```

Comment:
At what point does it describe how to download and install the new version?

Reply:
Done as suggested 👍